### PR TITLE
95 jsonbcreator quickfix

### LINKED ISSUE #95: JsonbCreator quick fix and automated tests
- Add a quick-fix code action to remove additional `@JsonbCreator` annotations associated with this diagnostic: https://github.com/eclipse/lsp4jakarta/issues/79. See the `DeleteAnnotationProposal` class included in this pull request for reference: https://github.com/eclipse/lsp4jakarta/pull/87

- Add automated tests for the above diagnostic and quick-fix. See [JakartaServletTest](https://github.com/eclipse/lsp4jakarta/blob/master/jakarta-eclipse/org.eclipse.lsp4jakarta.tests/src/main/java/org/eclipse/lsp4jakarta/jdt/servlet/JakartaServletTest.java) for an example

### PR DESCRIPTION
**Pull request by lamminade**
Fixes #95 

Added quickfix for jsonbcreator errors to add a "Remove annotation" quickfix to each jsonbcreator annotation (there should only be one)

Added a code, source, and severity to the diagnostic.

Also added tests for the diagnostic and quickfix

### CODE REVIEW

File: jakarta-eclipse/org.eclipse.lsp4jakarta.tests/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/ExtraJsonbCreatorAnnotations.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.sample.jakarta.jsonb;
+
+import jakarta.json.bind.annotation.JsonbCreator;
+
+public class ExtraJsonbCreatorAnnotation {
```
Comment (kathrynkodama) [inline jakarta-eclipse/org.eclipse.lsp4jakarta.tests/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/ExtraJsonbCreatorAnnotations.java:17]: class name is missing `s`:  ExtraJsonbCreatorAnnotations, class can be renamed to singular or class name and associated constructors should be updated

**Review by kathrynkodama — COMMENTED** [review]
Thank you @lamminade, looks great! Just one small fix

File: jakarta-eclipse/org.eclipse.lsp4jakarta.tests/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/ExtraJsonbCreatorAnnotations.java (hunk 1 above)
Comment (lamminade) [reply jakarta-eclipse/org.eclipse.lsp4jakarta.tests/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/ExtraJsonbCreatorAnnotations.java]: 371ec55

**Review by kathrynkodama — APPROVED** [review]
Looks good to me, thanks @lamminade!

**Pull request merged**